lib/halfnarp.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
```

Comment:
I think this is only needed once for 33c3. The majority of conferences will not have halfnarp data.